### ISSUE TITLE
FYI - Laravel 7 was released today - how this impacts our studies

### ISSUE THREAD

**Issue by susanBuck** [susanBuck/e15-spring20#17]
Hi everyone,

Just an FYI that Laravel 7 was released today ([release notes...](https://laravel.com/docs/7.x/releases#laravel-7))

I wasn't able to address this in the lecture videos I'm posting this evening (Week 6), so **I'll address it in next week's lectures (Week 7)**.

In the meantime, you'll note that when you go to the Docs (<https://laravel.com/docs>) it'll now default to version 7.x (<https://laravel.com/docs/7.x>). You can easily switch it back to v6 from the dropdown on the top right. 

That being said, most of the updates coming with v7 relate to features we're not yet working with, so you shouldn't notice a difference between the two versions for the content we're studying.

All of your existing applications (bookmark, p2) written in v6 will continue to function just as they have been previously. Next week I'll show you how they can be updated, and talk in general about how to manage framework updates.

**Comment by kreinholz** [comment]
Nice! Call me a glutton for punishment, but I get excited about updates. I remember being wished "good luck" in CSCI E-31 when I jumped on the Node.js 13.x bandwagon as soon as it was released (we started and at least officially stayed on 12.x for the duration of the semester), but didn't experience any problems.

With a robust framework like Laravel, I would hope/expect even major version updates wouldn't "break" things too badly or make upgrading too painful. Now AngularJS to Angular 2.x and all subsequent versions, or Python 2.x to Python 3.x, yeah, but generally updates are good things, right? :)
